client/my-sites/domains/domain-management/style.scss

```diff
@@ -267,6 +267,15 @@ input[type='radio'].domain-management-list-item__radio {
 	.form-toggle__label-content {
 		margin-left: 0;
 	}
+
+	.form-toggle:checked + .form-toggle__label .form-toggle__switch,
```

Comment:
We can just remove this, and use the default one, but added it here for simplicity.